prog/prog.go

```diff
@@ -49,6 +49,7 @@ const (
 	ArgGroup // logical group of args (struct or array)
 	ArgUnion
 	ArgReturn // fake value denoting syscall return value
+	ArgCsum
```

Comment:
Do we really need to introduce another arg type? I would strongly prefer if we don't grow this list, it just adds complexity everywhere. It seems that ArgCsum plays absolutely minimal role. It's really no more than a const with 0 value.